src/Zynga/Framework/IO/Disk/V1/ManagerTest.hh

```diff
@@ -87,14 +87,6 @@ class ManagerTest extends TestCase {
     $this->assertEquals($result, 4);
   }
 
-  public function testFwriteWithInvalidHandleReturnsFalse(): void {
-    $class = new ReflectionClass('Zynga\Framework\IO\Disk\V1\Manager');
-    $method = $class->getMethod('fwrite');
-    $method->setAccessible(true);
-    $result = $method->invoke(DiskIOManager::instance(), null, 'test');
-    $this->assertFalse($result);
-  }
-
   public function testFCloseWithValidHanldeReturnsTrue(): void {
     mkdir('/home/deploy/tmp/ManagerTest/6', 0777, true);
```

Comment:
Sorry, I missed this earlier (didn't get a chance to get through the tests).

Instead of these specific paths, you can tmpfile and  sys_get_temp_dir to get the OS paths for temporary files. That way, you don't have to hard code these paths and it becomes more portable. 

```
// This will let you create a tmp file with some name that you can mess with.
 $tmpFileName = tempnam(sys_get_temp_dir(),'');
chmod(tmpFileName, 0777);

// Rest of your code.